editor/plugins/gpu_particles_2d_editor_plugin.cpp

```diff
@@ -57,6 +57,31 @@ void GPUParticles2DEditorPlugin::_file_selected(const String &p_file) {
 	emission_mask->popup_centered();
 }
 
+void GPUParticles2DEditorPlugin::_selection_changed() {
+	List<Node *> selected_nodes = editor->get_editor_selection()->get_selected_node_list();
+
+	if (selected_particles.is_empty() && selected_nodes.is_empty()) {
+		return;
+	}
+
+	for (GPUParticles2D *SP : selected_particles) {
+		print_line(SP->get_name());
```

Comment:
This will be spammy.

Reply:
If you talk about the print_line() - that was forgot ... and removed.